internal/clients/flag/flag.go

```diff
@@ -14,12 +22,24 @@ type Values struct {
 }
 
 // Parse parses flags from command line
-func Parse() *Values {
+func Parse() (*Values, bool) {
+	fs := pflag.NewFlagSet("all flags", pflag.ContinueOnError)
```

Comment:
A flag set needs to be used so that we can set `ErrorHandling = pflag.ContinueOnError` which allows us to catch the parsing error and handle it appropriately.